### PR TITLE
[Image Resizing] Add Next.js next/image integration instructions

### LINKED ISSUE #672: [Image Resizing] Add documentation example showing how to use next/image
Unfortunately, [they aren't taking any PRs to add additional built-in loaders](https://github.com/vercel/next.js/pull/21443#issuecomment-765506401), but `next/image` is a popular component capable of using external resizing services to render an image.

Happy to do the PR if it's of interest. It should just be what follows, but I haven't properly tested yet.

```typescript
import Image from 'next/image'

const normalizeSrc = (src: string): string => {
  return src[0] === '/' ? src.slice(1) : src
}

const cloudflareLoader = ({ src, width, quality }) => {
  const params = [`width=${width}`]
  if (quality) {
    params.push(`quality=${quality}`)
  }
  const paramsString = params.join(',')
  return `/cdn-cgi/image/${paramsString}/${normalizeSrc(src)}`
}

const MyImage = (props) => {
  return (
    <Image
      loader={myLoader}
      src="/me.png"
      alt="Picture of the author"
      width={500}
      height={500}
    />
  )
}
```

### PR DESCRIPTION
**Pull request by GregBrimble**
Fixes #672.

### CODE REVIEW

**Comment by GregBrimble** [comment]
cc. @kodster28 @signalnerve

**Comment by kodster28** [comment]
Thanks for the PR @GregBrimble. I've also emailed the relevant PMs to review the technical details.

**Review by kristianfreeman — APPROVED** [review]

**Comment by GregBrimble** [comment]
Sorry, just spotted a tiny mistake on one of the lines where I had left in some TypeScript typings. Taken those out now :)

**Review by YoutacRSVARs — APPROVED** [review]

**Review by YoutacRSVARs — APPROVED** [review]

**Review by YoutacRSVARs — APPROVED** [review]

File: products/images/src/content/integration-with-frameworks.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Integration with frameworks
+order: 7
+---
+
+# Integration with frameworks
+
+## Next.js
+
+Image Resizing can be used automatically with Next.js' [`next/image` component](https://nextjs.org/docs/api-reference/next/image). With a [custom loader](https://nextjs.org/docs/api-reference/next/image#loader) which applies Cloudflare Image Resizing, `next/image` will set an optimal width and quality for a given client.
```
Comment (kodster28) [inline products/images/src/content/integration-with-frameworks.md:10]: Small wording tweaks: 

- Change to `automatically with the Next.js...` (remove apostrophe, add the)
- Change `which applies` to `applying`

**Review by kodster28 — CHANGES_REQUESTED** [review]

**Review by YoutacRSVARs — APPROVED** [review]

**Review by YoutacRSVARs — APPROVED** [review]

**Review by kodster28 — APPROVED** [review]

**Pull request merged**